modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -254,7 +254,7 @@ func (suite *KeeperTestSuite) TestChanUpgradeTry() {
 				channel.UpgradeSequence = 5
 				path.EndpointB.SetChannel(channel)
 			},
-			types.NewUpgradeError(6, types.ErrInvalidUpgradeSequence), // max sequence + 1 will be returned
+			types.NewUpgradeError(5, types.ErrInvalidUpgradeSequence), // channel sequence - 1 will be returned
```

Comment:
slightly confused me here cause I was looking at 5 in 254 and then wondering why is the sequence in the upgrade error 5 too! (might be worth noting in comment that upgrade sequence is bumped in init as this is non-crossing hellos case)